src/style/CAREUI.css

```
@@ -102,8 +102,8 @@
 .button-primary-ghost { @apply accent-primary-500 hover:bg-primary-100 text-primary-500 !important }
 .button-primary-border { @apply border border-primary-500 }
 
-.button-secondary-default { @apply accent-secondary-200 bg-white hover:bg-secondary-200 text-secondary-800 !important }
-.button-secondary-ghost { @apply accent-secondary-200 hover:bg-secondary-100 text-secondary-700 !important }
+.button-secondary-default { @apply accent-secondary-300 bg-secondary-300 hover:bg-secondary-200 text-secondary-800 !important }
```

@Ashesh3 this was intentionally kept white. How about reverting the changes for the default type, but preserving your changes for the ghost type alone?

```suggestion
.button-secondary-default { @apply accent-secondary-300 bg-secondary-300 hover:bg-secondary-200 text-secondary-800 !important }
```